scratch/Functional_Performance/Snakefile_functional_performance.smk

```diff
@@ -0,0 +1,31 @@
+import sys
+sys.path.insert(0, 'C:\\Users\\ggorski\\OneDrive - DOI\\USGS_ML\\DO\\drb-do-ml\\scratch\\Functional_Performance\\src')
```

Comment:
I'm curious why you changed this to an absolute path. I think we will need the relative path here. Did `"src"` not work?

Reply:
This is mostly for my own sanity, I am working on two sets of it functions, one for DO and one for Estuary Salinity, they rely on the same code but the wrapper functions are slightly different, so I just want to make sure that I am loading the correct module